MANIFEST.in

```diff
@@ -3,7 +3,7 @@ include setup*
 include README*
 include CHANGES.md
 include abstract.png
-include version.txt
+include spinalcordtoolbox/version.txt
```

Comment:
I did some testing and found that nothing in this file was being respected. I don't know if it ever was. According to my experiments any file outside of a package (as defined in setup.py:

>     packages=find_packages(exclude=['.git', 'data', 'dev', 'dev.*', 'install', 'testing']),

) is just ignored.

This patch here does do something but it's the only line that does.

An equivalent patch, I believe, would be

```
(.venv) [kousu@requiem spinalcordtoolbox]$ git diff
diff --git a/setup.py b/setup.py
index 8057fa56..0e808c41 100644
--- a/setup.py
+++ b/setup.py
@@ -39,6 +39,9 @@ setup(
     packages=find_packages(exclude=['.git', 'data', 'dev', 'dev.*',
                                     'install', 'testing']),
     include_package_data=True,
+    package_data={
+        "spinalcordtoolbox": ["version.txt"],
+    },
     extras_require={
         'docs': [
         'sphinx',
``` 

+

`git rm MANIFEST.in`